src/test/java/org/apache/commons/lang3/time/DateUtilsTest.java

```diff
@@ -693,6 +693,43 @@ public void testToCalendar() {
             // expected
         }
     }
+    
+    //-----------------------------------------------------------------------
+    @Test
+    public void testToCalendarWithDate() {
+        assertEquals("Convert Date and TimeZone to a Calendar, but failed to get the Date back", date1, DateUtils.toCalendar(date1, zone).getTime());
+        try {
+            DateUtils.toCalendar(null, zone);
+            fail("Expected NullPointerException to be thrown");
+        } catch(final NullPointerException npe) {
+            // expected
+        }
+    }
+    
+    //-----------------------------------------------------------------------
+    @Test
+    public void testToCalendarWithTimeZone() {
+    	assertEquals("Convert Date and TimeZone to a Calendar, but failed to get the TimeZone back", zone, DateUtils.toCalendar(date1, zone).getTimeZone());
+        try {
+            DateUtils.toCalendar(date1, null);
+            fail("Expected NullPointerException to be thrown");
+        } catch(final NullPointerException npe) {
+            // expected
+        }
+    }
+    
+  //-----------------------------------------------------------------------
+    @Test
+    public void testToCalendarWithDateAndTimeZone() {
+        try {
+        	Calendar c = DateUtils.toCalendar(date2, defaultZone);
+        	assertEquals("Convert Date and TimeZone to a Calendar, but failed to get the Date back", date2, c.getTime());
+        	assertEquals("Convert Date and TimeZone to a Calendar, but failed to get the TimeZone back", defaultZone, c.getTimeZone());
```

Comment:
This looks redundant to me, since `testToCalendarWithDate()` already tests that the date has been set and `testToCalendarWithTimeZone()` already tests, that the timeZone has been set.